database/migrations/0040-add_contribution_link_id_to_user.ts

```diff
@@ -0,0 +1,14 @@
+/* MIGRATION TO ADD contribution_link_id FIELD TO users */
+
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+
+export async function upgrade(queryFn: (query: string, values?: any[]) => Promise<Array<any>>) {
+  await queryFn(
+    'ALTER TABLE `users` ADD COLUMN `contribution_link_id` int UNSIGNED DEFAULT NULL AFTER `referrer_id`;',
```

Comment:
It has to be added a userId to the contributionLink Table, if such a relation will be necessary.